client/actions.js

```diff
@@ -88,6 +88,14 @@ export const CHAT_UPDATE_USER_OFFLINE = 'CHAT_UPDATE_USER_OFFLINE'
 export const DIALOG_OPEN = 'DIALOG_OPEN'
 export const DIALOG_CLOSE = 'DIALOG_CLOSE'
 
+// We have clicked on a folder inside the file browser
+export const FILE_BROWSER_CHANGE_PATH = 'FILE_BROWSER_CHANGE_PATH'
+// We are starting the process of getting the list of files (and folders) from a user's computer
+export const FILE_BROWSER_GET_BEGIN = 'FILE_BROWSER_GET_BEGIN'
+// We have received a response to our files retrieval (error or the list of the files for that
+// particular folder)
+export const FILE_BROWSER_GET = 'FILE_BROWSER_GET'
```

Comment:
feel like it's a bit odd to just have an action named `GET` with this prefix, maybe `GET_FILE_LIST` or `GET_LIST` or something along those lines?